addons/payment_stripe/models/payment_acquirer.py

```diff
@@ -108,7 +108,13 @@ def action_stripe_connect_account(self, menu_id=None):
             connected_account = self._stripe_fetch_or_create_connected_account()
 
             # Link generation
-            menu_id = menu_id or self.env.ref('payment.payment_acquirer_menu').id
+            if not menu_id:
+                # Fall back on account payment menu
+                menu = self.env.ref('account_payment.payment_acquirer_menu', False)
```

Comment:
What is it that changed and now requires a fallback on `account_payment`'s menu first?